spec/annexes.html

```diff
@@ -25,10 +25,10 @@
           The set of available locales for each constructor (<emu-xref href="#sec-internal-slots"></emu-xref>)
         </li>
         <li>
-          The BestFitMatcher algorithm (<emu-xref href="#sec-BestFitMatcher"></emu-xref>)
+          The BestFitMatcher algorithm (<emu-xref href="#sec-bestfitmatcher"></emu-xref>)
```

Comment:
http://bterlson.github.io/ecmarkup/#emu-xref because you can have a text, instead of the index value.